docs/package.json

```diff
@@ -16,7 +16,7 @@
   "dependencies": {
     "css-loader": "^0.23.0",
     "highlight.js": "^8.9.1",
-    "history": "^1.11.1",
+    "history": "1.13.0",
```

Comment:
Yeah that was weird - I had a merge conflict (didn't intend to commit that change), so aborted the PR, fixed it up and came back to find your comments. Don't know how the PR ever completed. Gremlins!